tests/test_invalidtoken/src/main.xc

```diff
@@ -1,33 +1,29 @@
 // Copyright 2016-2021 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
-/*
- * Test the use of the ExampleTestbench. Test that the value 0 and 1 can be sent
- * in both directions between the ports.
- *
- * NOTE: The src/testbenches/ExampleTestbench must have been compiled for this to run without error.
- *
- */
+
 #include <xs1.h>
 #include <print.h>
 #include <stdio.h>
 #include "xud.h"
 #include "platform.h"
-//#include "test.h"
-#include "xc_ptr.h"
 
-//#error
+#define EP_COUNT_OUT   (6)
+#define EP_COUNT_IN    (6)
 
-#define XUD_EP_COUNT_OUT   5
-#define XUD_EP_COUNT_IN    5
 
-//extern xc_ptr char_array_to_xc_ptr(const unsigned char a[]);
+XUD_EpType epTypeTableOut[EP_COUNT_OUT] = {XUD_EPTYPE_CTL, 
+                                            XUD_EPTYPE_BUL,
+                                            XUD_EPTYPE_BUL,
+                                            XUD_EPTYPE_BUL,
+                                            XUD_EPTYPE_BUL,
+                                            XUD_EPTYPE_BUL};
 
-/* Endpoint type tables */
-XUD_EpType epTypeTableOut[XUD_EP_COUNT_OUT] = {XUD_EPTYPE_CTL, XUD_EPTYPE_BUL,
-                                                XUD_EPTYPE_ISO,
-                                                XUD_EPTYPE_BUL,
-                                                 XUD_EPTYPE_BUL};
-XUD_EpType epTypeTableIn[XUD_EP_COUNT_IN] =   {XUD_EPTYPE_CTL, XUD_EPTYPE_BUL, XUD_EPTYPE_ISO, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL};
+XUD_EpType epTypeTableIn[EP_COUNT_IN] =   {XUD_EPTYPE_CTL, 
+                                            XUD_EPTYPE_BUL, 
```

Comment:
This line and a few after it have trailing whitespace.

Reply:
Fixed